.github/workflows/ci.yml

```
@@ -0,0 +1,41 @@
+name: Continuous Integration
+
+on: ['push', 'pull_request']
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        elixir: [1.9.x, 1.8.x, 1.7.x]
+        otp: [22.3, 21.2, 18.3]
```

```suggestion
        otp: [22.3, 21.2, 20.2]
```

Very important! Erlang/OTP is not supported by Elixir 1.7. The support of Elixir 1.7 is for Erlang versions between the v19 and the v22. The latest Elixir that supports Erlang/OTP v18 was Elixir 1.5!

So I have two ideas:

  - Remove the 18.3
  - Replace with a supported version of Erlang/OTP available on Ubuntu Packages repository.

In this suggestion I suggest to apply the second option, due it is available a v20.2 version of Erlang/OTP.